INSTALL.rst

```diff
@@ -300,6 +311,10 @@ To enable the null backend, make sure **[extensions]** contains::
 
   haas.ext.auth.null =
 
+Keystone Backend
```

Comment:
It may make sense to place this somewhere before `Null`, since that one is only for dev/testing